docs/getting-started/ory-cloud-oauth2.mdx

```diff
@@ -0,0 +1,213 @@
+---
+id: ory-cloud-oauth2
+title: Perform OAuth2 Authorization Code Grant with Ory Cloud
+sidebar_label: OAuth2 in Ory Cloud
+---
+
+This guide shows you a sample implementation of an Authorization Code Grant OAuth2 flow in Ory Cloud that you can use as a
```

Comment:
Can we add some context on

* What OAuth2 is for (3rd party API access, signing into 3rd party apps using your Ory identities ["becoming a social login provider"]
* The fact that Ory Cloud supports this out-of-the box incl. OIDC, Authorization Code Flows, Client Credentials Grant etc.
* A pointer to the in-depth-guides to the concepts in hydra docs (https://www.ory.sh/docs/hydra/concepts/before-oauth2)
* This guide will lead the user end-to-end through a first set-up, with the example of an Authorization Code Grant flow


Thanks!